File: java/oop/InterfaceApp/src/com/techlabs/icrudable/CustomerDb.java
package com.techlabs.icrudable;

public class CustomerDb implements Icrudable {

	@Override
	public void create() {
		System.out.println("Customer's create operation");
		
	}

	@Override
	public void read() {
		System.out.println("Customer's create operation");
	}

	@Override
	public void update() {
		System.out.println("Customer's update operation");
		
	}

	@Override
	public void remove() {
		System.out.println("Customer's remove method");
		
	}
	

}

File: dotNet/csharp/InterfaceCrudApp/InterfaceCrudApp/CustomerDb.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace InterfaceCrudApp
{
    class CustomerDb : Icrudable
    {
        public void Create()
        {
            Console.WriteLine("Customer's create operation");
        }

        public void Read()
        {
            Console.WriteLine("Customer's Read operation");
        }

        public void Remove()
        {
            Console.WriteLine("Customer's Remove operation");
        }

        public void Update()
        {
            Console.WriteLine("Customer's Update operation");
        }
    }
}
